01-messaging.md

```diff
@@ -95,30 +95,30 @@ connection.
 The sending node SHOULD use the minimum lengths required to represent
 the feature fields.  
 
-The sender MUST set feature bits as defined in [BOLT #9](09-features.md),
+The sender MUST set feature bits as defined in [BOLT #9](09-features.md)
 and MUST set to zero any feature bits that are not defined.
 
 The receiver MUST respond to known feature bits as specified in
-[BOLT #9](09-features.md).  For unknown feature bits which are
-non-zero, the receiver MUST ignore the bit if the bit number is odd,
+[BOLT #9](09-features.md). For unknown feature bits which are
+non-zero, the receiver MUST ignore the bit if the bit number is odd
 and MUST fail the connection if the bit number is even.
 
 Each node MUST wait to receive `init` before sending any other messages.
 
 #### Rationale
 
-This semantic allows future incompatible changes, or backward
-compatible changes.  Bits should generally be assigned in pairs, so
+This semantic allows future incompatible changes and/or backward
+compatible changes. Bits should generally be assigned in pairs, so
```

Comment:
How about:

    This semantic allows both future incompatible changes and future backward compatible changes.

ie. odd bits->cool if you don't understand, even bits->error if you don't understand.

Reply:
Sounds good.